contracts/StablecoinConverter.sol

```diff
@@ -499,7 +499,7 @@ contract StablecoinConverter is EpochTokenLocker {
             getBalance(user, tokenIdToAddressMap(order.sellToken))
         );
         uint256 limitTerm = currentPrices[order.sellToken].mul(order.priceDenominator)
-            .sub(currentPrices[order.buyToken].mul(order.priceNumerator));
+            .sub(order.priceNumerator.mul(feeDenominator).div(feeDenominator-1).mul(currentPrices[order.buyToken]));
         return leftoverSellAmount.mul(limitTerm).div(order.priceDenominator).toUint128();
```

Comment:
I noticed that we are adding a fee to the `buy_amount * buy_price` instead of removing a fee from the `sell_price * sell_amount`. I don't think this would cause any issues but I'm just mentioning it because I think that in #276 it was the other way around.

Reply:
Yes, this is how @fleupold got around the subtraction underflows that were happening.

Reply:
@fleupold Actually, I just realized there might be an issue with this:
`(buy_amount * fee_denominator) / (fee_denominator - 1)` introduces some error (0.0 .. 0.998 in this case) in the division which gets multiplied by `buy_price`. This means we can get up to `(buy_price * 998) / 1000` error in that term. I think the division should move to the end.

Reply:
>  I think the division should move to the end.

As in move it to the last operation.

Reply:
That works for me